src/components/Select/Select.tsx

```diff
@@ -31,6 +33,7 @@ export type SelectProps = Omit<
   shiftIconLeftwards?: boolean;
   value?: string | undefined;
   dataTestId?: string;
+  color?: TextColor;
```

Comment:
I think naming this `colorSwatch` is a bit more clear, cause `color` may make you think that this defines the color of the whole component.

Reply:
Ok, that sounds good.
If it changes the color of the text, it makes sense to name it `color`.
If it changes the background color, it makes sense to name it `backgroundColor`.
If it adds a color swatch, it makes sense to name it `colorSwatch`.